doorstop/core/tests/test_all.py

```diff
@@ -575,6 +586,14 @@ def setUp(self):
         self.tree = core.build(cwd=FILES, root=FILES)
         self.document = self.tree.find_document("REQ")
         self.temp = tempfile.mkdtemp()
+        # Use local plantuml of known version
+        plantuml_cmd = "java -Dplantuml.include.path=includes -jar " + os.path.join(
```

Comment:
I don't think we want to make `java` a dependency of this library. Can you solve this with a Python library?